tests/test_robot_states_flatcache.py

```diff
@@ -114,3 +116,88 @@ def camera_pose_test(flatcache):
 
 def test_camera_pose_flatcache_on():
     camera_pose_test(True)
+
+
+def test_robot_load_drive():
+    if og.sim is None:
+        # Set global flags
+        gm.ENABLE_OBJECT_STATES = True
+        gm.USE_GPU_DYNAMICS = True
+        gm.ENABLE_TRANSITION_RULES = False
+    else:
+        # Make sure sim is stopped
+        og.sim.stop()
+
+    config = {
+        "scene": {
+            "type": "Scene",
+        },
+    }
+
+    env = og.Environment(configs=config)
```

Comment:
I think you should add `og.sim.stop()` here, and then remove the `og.sim.stop()` in line 145.